ext/session/tests/user_session_module/bug31454.phpt

```diff
@@ -21,6 +21,6 @@ try {
 echo "Done\n";
 ?>
 --EXPECTF--
-Deprecated: Calling session_set_save_handler() with more than 2 arguments is deprecated in %s on line %d
+Deprecated: session_set_save_handler(): Calling session_set_save_handler() with more than 2 arguments is deprecated in %s on line %d
```

Comment:
Maybe we should change the actual error message to avoid the double use of `session_set_save_handler()`?